src/pages/index.js

```diff
@@ -1,84 +1,92 @@
-    //altText: "A view over a low mountain range facing the setting sun.",
-  },
+const handleImageClick = (title, link) => {
+  cardImageModal.openModal(title, link);
+};
 
-  {
-    title: "Latemar",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/latemar.jpg",
-    //altText: "A starry night view of three stone mountain peaks.",
-  },
+const handleDeleteClick = (card, cardId) => {
+  deleteCardConfirm.openModal(card, cardId);
+};
 
-  {
-    title: "Vanoise National Park",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/vanoise.jpg",
-    //altText:
-    // "A view of a rock beach near a calm lake, reflecting the stony mountains in the distance.",
-  },
+const handleLikeClick = (cardId, isLiked) => {
+  return api.updateCardLikes(cardId, isLiked).catch((err) => {
+    console.error(err);
+  });
+};
 
+const defaultDestinationSection = new Section(
   {
-    title: "Lago di Braies",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/lago.jpg",
-    //altText:
-    // "A view of a series of row boats tied to a wharf on a lake surrounded by mountains.",
+    renderer: renderCard,
   },
-];
-
-// const modalImage = document.querySelector("#modal-image");
-const cardTemplate = document.querySelector("#card-template");
-// const cardListEl = document.querySelector(".destinations");
-
-const addCardForm = document.querySelector("#profile-add-form");
-const editCardForm = document.querySelector("#profile-edit-form");
-
-const profileElement = document.querySelector(".profile");
-
-//const profileName = profileElement.querySelector(".profile__name");
-const profileNameSelector = ".profile__name";
-//const profileProfession = profileElement.querySelector(".profile__profession");
-const profileProfessionSelector = ".profile__profession";
-
-const openEditButton = document.querySelector(".profile__button-edit");
-const professionInput = document.querySelector(".modal__input-profession");
-const nameInput = document.querySelector(".modal__input-name");
-//const titleInput = document.querySelector(".modal__input-title");
-//const imageInput = document.querySelector(".modal__input-image");
-//const cardListEl = document.querySelector(".destinations");
-const destinations = ".destinations";
+  destinations
+);
 
-//const viewImageModal = document.querySelector("#viewImage-modal");
+// API promise to load cards and profile information
+Promise.all([api.getUserInfo(), api.getInitialCards()])
+  .then(([userData, cards]) => {
+    userInfo.setUserInfo({ userData });
+    userInfo.setAvatar(userData.avatar);
 
-const openAddButton = document.querySelector(".profile__button-add");
+    defaultDestinationSection.renderItems(cards);
+  })
+  .catch(console.error);
 
+// Creates a singular location for the various classes involved in validation
 const formValidationConfig = {
```

Comment:
From the 8th sprint all constants should be in file `constants.js` in folder `utils` to clean up the main file.

Reply:
formValidationConfig has been moved to constants.